pyvisa-py/gpib.py

```diff
@@ -165,6 +171,42 @@ def write(self, data):
             else:
                 return 0, StatusCode.error_system_error
 
+    def clear(self):
+        """Clears a device.
+
+        Corresponds to viClear function of the VISA library.
+
+        :param session: Unique logical identifier to a session.
+        :return: return value of the library call.
+        :rtype: :class:`pyvisa.constants.StatusCode`
+        """
+
+        logger.debug('GPIB.device clear')
+
+        try:
+            self.interface.clear()
+            return SUCCESS
```

Comment:
SUCCESS does not appear to be defined anywhere.

Reply:
Oh yeah... this was a leftover from the previous stable release of pyvisa-py:
https://github.com/pyvisa/pyvisa-py/blob/7f408571bb2c389e361ecc356fa4e3565365986b/pyvisa-py/gpib.py#L40  
fixed by dc3d4b0